swagger.yml

```diff
@@ -2284,10 +2283,10 @@ paths:
           name: run_id
           required: true
           type: string
-        - in: query
-          name: before
-          type: string
-          format: date-time
+#        - in: query
```

Comment:
No `after` instead?

Reply:
The one I wanted to discuss. In order to have pagination on a specific run's tasks, we need to have the combination of action name and hook it passed as a token or have run_id/action_name/hook_id and have pagination cross runs.